### DISCUSSION TITLE
Edit multiple parts at once

### DISCUSSION THREAD

**Discussion by new55j** [Part-DB/Part-DB-server#525] [Ideas]
Thank you very much for the intensive work on PartDB Server, I really appreciate this program.

I would like to delete a whole series of parts at once.

In the table view, you can select several parts by ticking them. The only thing you can do is favor parts and set the review status. It would be a great advantage to be able to edit several parts in other ways, for example delete, move to another category, add parameters...

**Comment by jbtronics** [comment]
> I would like to delete a whole series of parts at once.

That is actually already possible.



> In the table view, you can select several parts by ticking them. The only thing you can do is favor parts and set the review status

If you scroll down in that menu, you should be able to select other kind of operations like "Delete", "Export", "Label generation" or mass changing the category, footprint, manufacturer or part unit.

**Reply by new55j** [reply]
That's exactly what I meant... I didn't find it right away. cheers

**Reply by Sebigamer4** [reply]
> > I would like to delete a whole series of parts at once.
> 
> That is actually already possible.
> 
> > In the table view, you can select several parts by ticking them. The only thing you can do is favor parts and set the review status
> 
> If you scroll down in that menu, you should be able to select other kind of operations like "Delete", "Export", "Label generation" or mass changing the category, footprint, manufacturer or part unit.

Thanks, i also searched for an hour before finding this and noticing that scrollbar
